lua/user/plugins/extras/editor/copilot.lua

```diff
@@ -3,6 +3,48 @@ return {
     "zbirenbaum/copilot.lua",
     event = "VeryLazy",
```

Comment:
i wonder when it would make sense to load it? on insert?

Reply:
Ah shit,  I was supposed to push the copilot changes to my own branch 🤣